regression/cbmc/dynamic-object-01/main.c

```diff
@@ -0,0 +1,10 @@
+#include <assert.h>
+#include <stdlib.h>
+
+void main()
+{
+  char *p = malloc(1);
+  free(p);
+
+  assert(!__CPROVER_DYNAMIC_OBJECT(p));
```

Comment:
to check it is havoc'd you should add an `assert(dynamic_object(p))`
```suggestion
  assert(!__CPROVER_DYNAMIC_OBJECT(p));
  assert(__CPROVER_DYNAMIC_OBJECT(p));
```

Reply:
Added